04-quality-control.md

```diff
@@ -8,6 +8,10 @@ Authors:**Christie Bahlai**, **Aleksandra Pawlik**<br>
 Contributors: **Jennifer Bryan**, **Alexander Duryee**, **Jeffrey Hollister**, **Daisie Huang**, **Owen Jones**,
 **Ben Marwick**, and **Ethan White**
 
+## Learning Objective 
+
+* Apply quality control techniques to ensure data in spreadsheets is free from errors.
+
```

Comment:
We can't ensure things are free from error, so maybe?

Apply quality control techniques to identify errors in spreadsheets and limit incorrect data entry